Add hint for Android Studio users to keep Android Gradle Plugin updated (#31032)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/optimizing-performance/build-cache/caching_android_projects.adoc

> == The Android Gradle Plugin and the Gradle Build Tool

Reviewer: ```suggestion
== The Android Gradle Plugin
```

Reviewer: You can use this to add DCO.

Reviewer: @Ayushcode10, go ahead and try to commit this with a sign off.

> To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).

> For Android Studio users: Ensure you are using the latest version of the Android Gradle Plugin to avoid compatibility issues and gain access to performance improvements with each new release.

Reviewer: ```suggestion
Android Studio users should use the latest Android Gradle Plugin to ensure compatibility and benefit from performance improvements in new releases.
```
